zipkin-storage/cassandra/src/test/resources/autocomplete_tags-stress.yaml

```diff
@@ -45,5 +45,5 @@ queries:
     cql: SELECT DISTINCT key FROM autocomplete_tags
```

Comment:
I *think* in these files we also want to update `select:  fixed(1)/1000`

Reply:
Yeah I saw those but I've no idea of what they mean. We can wait for the tests and see if they break

Reply:
Travis seems happy without that change

Reply:
travis doesn't execute these files.

Reply:
the stress tests are invoked manually I mean.

Reply:
How do I run them to test this?

Reply:
it says how at the top of the file.